README.md

```diff
@@ -15,7 +14,8 @@ Welcome to the Hyperledger Caliper project. Caliper is a blockchain performance
 
 Currently supported blockchain solutions:
 
-* [Hyperledger Fabric v1.X, v2.X](https://github.com/hyperledger/fabric)
+* [Hyperledger Fabric v2.X (Deprecated Network SDK, Gateway SDK)](https://github.com/hyperledger/fabric/tree/v2.5.13)
+* [Hyperledger Fabric v3.X (Gateway SDK)](https://github.com/hyperledger/fabric)
```

Comment:
I was trying to avoid claiming fabric 3.x support for a couple of reasons
1. We don't test it at all in the integration tests
2. Fabric 3.x is not LTS and I don't think whoever is left on Fabric cares about creating an LTS release (https://github.com/hyperledger/fabric/issues/5259)

Reply:
Fair enough, updated.